app-hitl/hitl/src/hitl/commands/run_hitl.py

```diff
@@ -1,4 +1,5 @@
 import json
+from lib2to3.pgen2 import driver
```

Comment:
@simonkassel I am not sure where this is from. It does not seem like it is used?

Reply:
done